realsense2_camera/CMakeLists.txt

```diff
@@ -163,7 +163,21 @@ add_library(${PROJECT_NAME} SHARED
     include/named_filter.h
     include/ros_param_backend.h
     include/profile_manager.h
-    
+    include/image_publisher.h)
+
+
+if (BUILD_TOOLS)
+
+  include_directories(tools/frame_latency/include)
```

Comment:
I don't know if I like this arrangement... I'd want to:
```
#include <tools/frame_latency/node.h>
```
I.e., all the tools' includes will be under one include directory settings.
Instead we have to add each tool's include directory manually to the include path...

Reply:
Since I don't want to place the tool includes under the wrapper include dir, I rearranged the frame_latency files so that the cpp need to do:
`#include <frame_latency/frame_latency.h>`

and on the CMakelist.exe it looks like this:
```
if (BUILD_TOOLS)

  include_directories(tools)
```

Is that acceptable?